### PR TITLE
Added Layout Builder and resolution

### LINKED ISSUE #5080: Modules and Plugins with Known Issues Doc Update Layout Builder
Re: [Modules and Plugins with Known Issues](https://pantheon.io/docsmodules-plugins-known-issues)

Priority: Low

## Issue Description:
This core module Layout Builder returns an error when trying to be activated `Fatal error: require(): Failed opening required '/srv/bindings/bindingIDxxxxxxxxx/code/vendor/composer/../../web/core/lib/Drupal/Core/Layout/Icon/SvgIconBuilder.php'` 

## Suggested Resolution
Outline steps on how to make the module work

### PR DESCRIPTION
**Pull request by carl-alberto**
Closes #5080

## Effect
PR includes the following changes:
-Outline steps on how to make the module work

## Remaining Work
- [ ] List any outstanding work here
- [ ] Technical review
- [x] Copy Review

## Post Launch
**Do not remove** - To be completed by the docs team upon merge:
- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] Update Status Report
- [ ] Archive from **Done** in [Main Project](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

File: source/content/modules-plugins-known-issues.md [hunk 1]
```diff
@@ -173,6 +173,18 @@ You can modify this patch according to your needs, such as performing an operati
...
+### [Layout Builder](https://www.drupal.org/docs/8/core/modules/layout-builder)
+**Issue**: This core module returns an error when trying activate:
+
+```php
+Fatal error: require(): Failed opening required '/srv/bindings/bindingIDxxxxxxxxx/code/vendor/composer/../../web/core/lib/Drupal/Core/Layout/Icon/SvgIconBuilder.php'
+```
+
+This is due to the .gitignore file that excludes `Icon/SvgIconBuilder.php`.
+
+**Solution**: In the `.gitignore` file in your local install or build server, remove the `Icon` line so that file can be included in your push or deploys.
```
Comment (alexfornuto) [inline source/content/modules-plugins-known-issues.md]: Can we replace `your local install or build server` with the relative path to the `.gitignore` file in the code base?

Comment (carl-alberto) [reply source/content/modules-plugins-known-issues.md]: Not really sure, whatever would be easiest to understand in a user's perspective as long as they can easily locate the .gitignore file and make sure the missing gets included in their repo and pushed to their codebase

Comment (alexfornuto) [reply source/content/modules-plugins-known-issues.md]: @carl-alberto what would that path be?

Comment (carl-alberto) [reply source/content/modules-plugins-known-issues.md]: @alexfornuto It should be in the root path of the Drupal installation (/code folder). But it seems the ones affected here are versions prior to 8.7.5  https://github.com/pantheon-systems/drops-8/commit/6af7bf3045b168a6fa4b866189bf143aa12f85af and latest D7 https://github.com/pantheon-systems/drops-7/commit/2eb2fd38ca57178fb138326dda4abb6329c815bc. Another alternative solution is to this is upgrade to the latest Drupal version

Comment (alexfornuto) [reply source/content/modules-plugins-known-issues.md]: @carl-alberto I'm asking for you to provide the path from the project root to the file in question to be removed.

Comment (carl-alberto) [reply source/content/modules-plugins-known-issues.md]: @alexfornuto If I'm not mistaken

/code/vendor/composer/../../web/core/lib/Drupal/Core/Layout/Icon/SvgIconBuilder.php

**Comment by alexfornuto** [comment]
Closing per discussion in #5080

**Pull request closed without merging**